### ISSUE TITLE
Container fill disappears

### ISSUE THREAD

**Issue by sixthtulip** [FlutterFlow/flutterflow-issues#2821]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

The parent container has a fill and a gradient. The child container has a TabBar widget and on Action Trigger Flip Animation. 
When I hover the mouse over the name of any of the tabs, the background fill disappears (becomes black). If I remove the Action Trigger Animation or Gradient the problem disappears. If Apply Initial State is disabled, then after the page loads, hovering over the tab names does not cause the background fill to disappear. But after playing the animation the problem occurs again. This problem only occurs with Flip animation.

### Expected Behavior

I expect the fill to not disappear on hover.

### Steps to Reproduce

1.Create a container on the page and place a child container in it
2. Place the TabBar widget in the child container
3.Set on Action Trigger Flip Animation for the child container.
4.Set the fill color and gradient for the parent container.
5.Start test mode and hover your mouse over the tab name.

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

ITESz/Hqx89gobRG+pDLKO16hQAjNkwmWLg7m9cbdAI0IoPmPIoXdur+VHteQseCdFA3G06Z+ToK0+zOjtrDVPACPRSWUoNo1Yd2Zj3zVj6ieLavErqCeXNAEflMJlyP5Ke7uSMkD7ZaL04Qw3z/cq3qNleeY8aSfxBlZ7vfcPo=

### Context

I'm trying to achieve expected behavior on a page.

### Visual documentation

https://github.com/FlutterFlow/flutterflow-issues/assets/168268647/5bf262c9-6dd0-491e-8789-25cd358b41db

### Environment

```markdown
- FlutterFlow version: 4.1.45+
- Platform: web
- Browser name and version: Chrome 124.0.6367.91
- Operating system and version affected: Windows 10
```

### General

- [ ] I was collaborating with other users in the same project prior to data loss
- [ ] I navigated between different project views leading up to data loss
- [ ] The UI was running very slowly prior to data loss
- [ ] A crash or other bug occurred prior to data loss
- [ ] Data was not saved in a snapshot
- [ ] I was using flutterflow from multiple devices
- [ ] I was using flutterflow from multiple platforms (e.g. both web and MacOS Desktop app)

### Relative to the time the changes were made, data was lost within

- [ ] 1 minute
- [ ] 1 hour
- [ ] 1 day
- [ ] 1 week

### When following my steps to reproduce, data loss happens

- [ ] ~100% of the time
- [ ] ~50% of the time
- [ ] ~10% of the time
- [ ] ~1% of the time
- [ ] Very rarely or only encountered once

**Comment by paulperez-dev** [comment]
Hi @sixthtulip! Thanks for submitting the issue. I was unable to reproduce the issue and I've taken a look at your project, but with a lot going on, it's challenging to pinpoint the issue. Given the nature of the problem you're highlighting, we need a blank project with the issue steps set up in it. Without these details, it might be a bit challenging for us to investigate the issue. Please provide the required information.

**Comment by sixthtulip** [comment]
Hello.

Recently I seem to have found the root of the problem.
We are talking about a parent container that has a gradient. The fact is that for one of these gradients the color is not specified, that is, it has transparency.

By the way, after a recent update, it became very difficult to set the color for the widget. Sliders do not respond well to movement.

And one more strange thing. Pay attention to the flip animation values. When the container is collapsed we see Initial is 0, final is 2. But if I expand the animation container I see that both values are 0.

Here is a clean project that I made for you. Launch it and hover your mouse over the tab name. You will see that the parent container's gradient has disappeared and the background has become black (although the page also has a non-black fill).

https://app.flutterflow.io/project/animation-bug-duz7fz

**Comment by paulperez-dev** [comment]
Hi @sixthtulip, I was able to reproduce the issue, I will send it to the ENG team so they can work on this!

**Comment by paulperez-dev** [comment]
Hi @sixthtulip, this was fixed on PR#1900. I will close this issue.

**Comment by mmearly** [comment]
@paulperez-dev This is not correct, only half the issue was fixed (specifically the final number of flips not working properly in the flip animation). The other problem of container fill disappearing still requires further investigation.
